AirSim on Unity failing compilation

I was trying to build AirSim for Unity following the steps in the Readme.md. 
Building AirSim with build.cmd worked well with the x64 VS 2017 cmd prompt but running Unity\build,cmd in the same cmd prompt failed. 🤔

It seems like the code is not really compatible with latest from master. I'm using 4414c8e37f1560bbe88dab56766d3827b52640d3 

Here's the full output:

```
D:\Git\OpenSource\Microsoft\AirSim\Unity>build.cmd

D:\Git\OpenSource\Microsoft\AirSim\Unity>REM //---------- copy binaries and include for MavLinkCom in deps ----------

D:\Git\OpenSource\Microsoft\AirSim\Unity>msbuild AirLibWrapper\AirsimWrapper.sln  /target:Clean /target:Build  /property:Configuration=Release /property:Platform=x64
Microsoft (R) Build Engine version 15.9.21+g9802d43bc3 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
Build started 18.09.2019 16:55:06.
Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper.sln" on node 1 (Clean;Build target(s)).
ValidateSolutionConfiguration:
  Building solution configuration "Release|x64".
Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper.sln" (1) is building "D:\Git\OpenSource\M
icrosoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (2) on node 1 (Clean target(s)).
Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (2) is bu
ilding "D:\Git\OpenSource\Microsoft\AirSim\AirLib\AirLib.vcxproj" (3) on node 1 (Clean target(s)).
_PrepareForClean:
  Deleting file "D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\AirLib.tlog\AirLib.lastbuildstate".
Project "D:\Git\OpenSource\Microsoft\AirSim\AirLib\AirLib.vcxproj" (3) is building "D:\Git\OpenSource\Microsoft\AirSim\
MavLinkCom\MavLinkCom.vcxproj" (4) on node 1 (Clean target(s)).
_PrepareForClean:
  Deleting file "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkCom.tlog\MavLinkCom.las
  tbuildstate".
Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\MavLinkCom.vcxproj" (Clean target(s)).

Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\AirLib\AirLib.vcxproj" (Clean target(s)).

Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (2) is bu
ilding "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj" (5) on node 1 (Clea
n target(s)).
_PrepareForClean:
  Deleting file "x64\Release\AirsimWrapper.tlog\AirsimWrapper.lastbuildstate".
Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj" (Cle
an target(s)).

Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metap
roj" (Clean target(s)).

Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper.sln" (1) is building "D:\Git\OpenSource\M
icrosoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (2:2) on node 1 (default targets).
Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (2:2) is
building "D:\Git\OpenSource\Microsoft\AirSim\AirLib\AirLib.vcxproj" (3:4) on node 1 (default targets).
Project "D:\Git\OpenSource\Microsoft\AirSim\AirLib\AirLib.vcxproj" (3:4) is building "D:\Git\OpenSource\Microsoft\AirSi
m\MavLinkCom\MavLinkCom.vcxproj" (4:2) on node 1 (default targets).
InitializeBuildStatus:
  Creating "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkCom.tlog\unsuccessfulbuild"
  because "AlwaysCreate" was specified.
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\CL.exe /c /I
  common_utils /Iinclude /ZI /JMC /nologo /W3 /WX /diagnostics:classic /Od /D _WIN32 /D _WIN32_WINNT=0x0600 /D NOMINMAX
   /D VC_EXTRALEAN /D _SCL_SECURE_NO_WARNINGS /D _CRT_SECURE_NO_WARNINGS /D _DEBUG /D _LIB /D _UNICODE /D UNICODE /Gm-
  /EHsc /RTC1 /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Fo"D:\Git\OpenSource\Microsoft\AirSim\MavLinkCo
  m\temp\MavLinkCom\x64\Debug\\" /Fd"D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkCom
  .pdb" /FR"D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\\" /Gd /TP /FC /errorReport:queue c
  ommon_utils\FileSystem.cpp common_utils\ThreadUtils.cpp src\UdpSocket.cpp src\impl\UdpSocketImpl.cpp src\AdHocConnect
  ion.cpp src\impl\AdHocConnectionImpl.cpp src\impl\MavLinkFtpClientImpl.cpp src\impl\MavLinkNodeImpl.cpp src\impl\MavL
  inkTcpServerImpl.cpp src\impl\onecore\OneCoreFindSerialPorts.cpp src\impl\windows\WindowsFindSerialPorts.cpp src\MavL
  inkFtpClient.cpp src\MavLinkLog.cpp src\MavLinkMessageBase.cpp src\MavLinkMessages.cpp src\MavLinkNode.cpp src\Semaph
  ore.cpp src\MavLinkTcpServer.cpp src\MavLinkVehicle.cpp src\impl\MavLinkVehicleImpl.cpp src\MavLinkConnection.cpp src
  \impl\MavLinkConnectionImpl.cpp src\MavLinkVideoStream.cpp src\impl\MavLinkVideoStreamImpl.cpp src\serial_com\SerialP
  ort.cpp src\serial_com\SocketInit.cpp src\serial_com\TcpClientPort.cpp src\serial_com\UdpClientPort.cpp src\serial_co
  m\wifi.cpp
  FileSystem.cpp
  ThreadUtils.cpp
  UdpSocket.cpp
  UdpSocketImpl.cpp
  AdHocConnection.cpp
  AdHocConnectionImpl.cpp
  MavLinkFtpClientImpl.cpp
  MavLinkNodeImpl.cpp
  MavLinkTcpServerImpl.cpp
  OneCoreFindSerialPorts.cpp
  WindowsFindSerialPorts.cpp
  MavLinkFtpClient.cpp
  MavLinkLog.cpp
  MavLinkMessageBase.cpp
  MavLinkMessages.cpp
  MavLinkNode.cpp
  Semaphore.cpp
  MavLinkTcpServer.cpp
  MavLinkVehicle.cpp
  MavLinkVehicleImpl.cpp
  Generating Code...
  Compiling...
  MavLinkConnection.cpp
  MavLinkConnectionImpl.cpp
  MavLinkVideoStream.cpp
  MavLinkVideoStreamImpl.cpp
  SerialPort.cpp
  SocketInit.cpp
  TcpClientPort.cpp
  UdpClientPort.cpp
  wifi.cpp
  Generating Code...
Lib:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\Lib.exe /OUT
  :"D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\lib\x64\Debug\MavLinkCom.lib" /LIBPATH:\stage\lib /NOLOGO /MACHINE:X6
  4   /machine:x64 D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\FileSystem.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\ThreadUtils.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\UdpSocket.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\UdpSocketImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\AdHocConnection.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\AdHocConnectionImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkFtpClientImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkNodeImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkTcpServerImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\OneCoreFindSerialPorts.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\WindowsFindSerialPorts.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkFtpClient.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkLog.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkMessageBase.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkMessages.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkNode.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\Semaphore.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkTcpServer.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVehicle.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVehicleImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkConnection.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkConnectionImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVideoStream.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVideoStreamImpl.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\SerialPort.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\SocketInit.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\TcpClientPort.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\UdpClientPort.obj
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\wifi.obj
  MavLinkCom.vcxproj -> D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\lib\x64\Debug\MavLinkCom.lib
BscMake:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x86\bscmake.exe
  /nologo /o"D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\lib\x64\Debug\MavLinkCom.bsc" /n D:\Git\OpenSource\Microsoft
  \AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\FileSystem.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLin
  kCom\x64\Debug\ThreadUtils.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\UdpSocket.sbr
  D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\UdpSocketImpl.sbr D:\Git\OpenSource\Microsoft
  \AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\AdHocConnection.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\M
  avLinkCom\x64\Debug\AdHocConnectionImpl.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\M
  avLinkFtpClientImpl.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkNodeImpl.sbr D
  :\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkTcpServerImpl.sbr D:\Git\OpenSource\Mic
  rosoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\OneCoreFindSerialPorts.sbr D:\Git\OpenSource\Microsoft\AirSim\MavL
  inkCom\temp\MavLinkCom\x64\Debug\WindowsFindSerialPorts.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLin
  kCom\x64\Debug\MavLinkFtpClient.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkLo
  g.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkMessageBase.sbr D:\Git\OpenSourc
  e\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkMessages.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLin
  kCom\temp\MavLinkCom\x64\Debug\MavLinkNode.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debu
  g\Semaphore.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkTcpServer.sbr D:\Git\O
  penSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVehicle.sbr D:\Git\OpenSource\Microsoft\AirSim
  \MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVehicleImpl.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLi
  nkCom\x64\Debug\MavLinkConnection.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLink
  ConnectionImpl.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVideoStream.sbr D:\
  Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkVideoStreamImpl.sbr D:\Git\OpenSource\Mic
  rosoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\SerialPort.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\
  MavLinkCom\x64\Debug\SocketInit.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\TcpClient
  Port.sbr D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\UdpClientPort.sbr D:\Git\OpenSource\
  Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\wifi.sbr
CopyFilesToOutputDirectory:
  Copying file from "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkCom.pdb" to "D:\Git
  \OpenSource\Microsoft\AirSim\MavLinkCom\lib\x64\Debug\MavLinkCom.pdb".
FinalizeBuildStatus:
  Deleting file "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkCom.tlog\unsuccessfulbu
  ild".
  Touching "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\temp\MavLinkCom\x64\Debug\MavLinkCom.tlog\MavLinkCom.lastbuil
  dstate".
Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\MavLinkCom\MavLinkCom.vcxproj" (default targets).

InitializeBuildStatus:
  Creating "D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\AirLib.tlog\unsuccessfulbuild" because "A
  lwaysCreate" was specified.
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\CL.exe /c /I
  include /Ideps\eigen3 /Ideps\rpclib\include /ID:\Git\OpenSource\Microsoft\AirSim\AirLib\..\MavLinkCom\include /Zi /no
  logo /W4 /WX /diagnostics:classic /O2 /Oi /GL /D NDEBUG /D _SCL_SECURE_NO_WARNINGS /D _CRT_SECURE_NO_WARNINGS /D _LIB
   /D _UNICODE /D UNICODE /Gm- /EHsc /MD /GS /Gy /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /Fo"D:\Git\OpenSource\
  Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\\" /Fd"D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Rele
  ase\AirLib.pdb" /Gd /TP /FC /errorReport:queue /w34263 /w34266 src\api\RpcLibClientBase.cpp src\api\RpcLibServerBase.
  cpp src\vehicles\multirotor\api\MultirotorApiBase.cpp src\safety\ObstacleMap.cpp src\safety\SafetyEval.cpp src\common
  \common_utils\FileSystem.cpp src\vehicles\car\api\CarRpcLibClient.cpp src\vehicles\car\api\CarRpcLibServer.cpp src\ve
  hicles\multirotor\api\MultirotorRpcLibClient.cpp src\vehicles\multirotor\api\MultirotorRpcLibServer.cpp
  RpcLibClientBase.cpp
  RpcLibServerBase.cpp
  MultirotorApiBase.cpp
  ObstacleMap.cpp
  SafetyEval.cpp
  FileSystem.cpp
  CarRpcLibClient.cpp
  CarRpcLibServer.cpp
  MultirotorRpcLibClient.cpp
  MultirotorRpcLibServer.cpp
Lib:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\Lib.exe /OUT
  :"D:\Git\OpenSource\Microsoft\AirSim\AirLib\lib\x64\Release\AirLib.lib" rpc.lib /LIBPATH:deps\rpclib\lib\x64\Release
  /NOLOGO /MACHINE:X64 /LTCG D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\RpcLibClientBase.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\RpcLibServerBase.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\MultirotorApiBase.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\ObstacleMap.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\SafetyEval.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\FileSystem.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\CarRpcLibClient.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\CarRpcLibServer.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\MultirotorRpcLibClient.obj
  D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\MultirotorRpcLibServer.obj
  AirLib.vcxproj -> D:\Git\OpenSource\Microsoft\AirSim\AirLib\lib\x64\Release\AirLib.lib
FinalizeBuildStatus:
  Deleting file "D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\AirLib.tlog\unsuccessfulbuild".
  Touching "D:\Git\OpenSource\Microsoft\AirSim\AirLib\temp\AirLib\x64\Release\AirLib.tlog\AirLib.lastbuildstate".
Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\AirLib\AirLib.vcxproj" (default targets).

Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (2:2) is
building "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj" (5:2) on node 1 (
default targets).
InitializeBuildStatus:
  Touching "x64\Release\AirsimWrapper.tlog\unsuccessfulbuild".
ClCompile:
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\CL.exe /c /I
  D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\..\..\..\AirLib\include\vehicles\car /ID:\Git\Op
  enSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\..\..\..\AirLib\include /ID:\Git\OpenSource\Microsoft\Air
  Sim\Unity\AirLibWrapper\AirsimWrapper\..\..\..\AirLib\deps\eigen3 /I"D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibW
  rapper\AirsimWrapper\..\..\..\external\rpclib\rpclib-2.2.1\include" /ID:\Git\OpenSource\Microsoft\AirSim\Unity\AirLib
  Wrapper\AirsimWrapper\..\..\..\MavLinkCom\include /ID:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrap
  per\include /Zi /W3 /WX- /diagnostics:classic /O2 /Oi /GL /D NDEBUG /D AIRSIMWRAPPER_EXPORTS /D _WINDOWS /D _USRDLL /
  D _WINDLL /D _UNICODE /D UNICODE /Gm- /EHsc /MD /GS /Gy /fp:precise /permissive- /Zc:wchar_t /Zc:forScope /Zc:inline
  /Fo"x64\Release\\" /Fd"x64\Release\vc141.pdb" /Gd /TP /FC /errorReport:queue Source\Logger.cpp Source\NedTransform.cp
  p Source\PawnSimApi.cpp Source\PInvokeWrapper.cpp Source\SimHUD\SimHUD.cpp Source\SimMode\SimModeBase.cpp Source\SimM
  ode\SimModeWorldBase.cpp Source\UnityImageCapture.cpp Source\UnitySensors\UnityDistanceSensor.cpp Source\UnitySensors
  \UnitySensorFactory.cpp Source\UnityToAirSimCalls.cpp Source\Vehicles\Car\CarPawn.cpp Source\Vehicles\Car\CarPawnApi.
  cpp Source\Vehicles\Car\CarPawnSimApi.cpp Source\Vehicles\Car\SimModeCar.cpp Source\Vehicles\Multirotor\FlyingPawn.cp
  p Source\Vehicles\Multirotor\MultirotorPawnEvents.cpp Source\Vehicles\Multirotor\MultirotorPawnSimApi.cpp Source\Vehi
  cles\Multirotor\SimModeWorldMultiRotor.cpp Source\WorldSimApi.cpp
  Microsoft (R) C/C++ Optimizing Compiler Version 19.16.27032.1 for x64
  Copyright (C) Microsoft Corporation.  All rights reserved.

  cl /c /ID:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\..\..\..\AirLib\include\vehicles\car /ID
  :\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\..\..\..\AirLib\include /ID:\Git\OpenSource\Micro
  soft\AirSim\Unity\AirLibWrapper\AirsimWrapper\..\..\..\AirLib\deps\eigen3 /I"D:\Git\OpenSource\Microsoft\AirSim\Unity
  \AirLibWrapper\AirsimWrapper\..\..\..\external\rpclib\rpclib-2.2.1\include" /ID:\Git\OpenSource\Microsoft\AirSim\Unit
  y\AirLibWrapper\AirsimWrapper\..\..\..\MavLinkCom\include /ID:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\Ai
  rsimWrapper\include /Zi /W3 /WX- /diagnostics:classic /O2 /Oi /GL /D NDEBUG /D AIRSIMWRAPPER_EXPORTS /D _WINDOWS /D _
  USRDLL /D _WINDLL /D _UNICODE /D UNICODE /Gm- /EHsc /MD /GS /Gy /fp:precise /permissive- /Zc:wchar_t /Zc:forScope /Zc
  :inline /Fo"x64\Release\\" /Fd"x64\Release\vc141.pdb" /Gd /TP /FC /errorReport:queue Source\Logger.cpp Source\NedTran
  sform.cpp Source\PawnSimApi.cpp Source\PInvokeWrapper.cpp Source\SimHUD\SimHUD.cpp Source\SimMode\SimModeBase.cpp Sou
  rce\SimMode\SimModeWorldBase.cpp Source\UnityImageCapture.cpp Source\UnitySensors\UnityDistanceSensor.cpp Source\Unit
  ySensors\UnitySensorFactory.cpp Source\UnityToAirSimCalls.cpp Source\Vehicles\Car\CarPawn.cpp Source\Vehicles\Car\Car
  PawnApi.cpp Source\Vehicles\Car\CarPawnSimApi.cpp Source\Vehicles\Car\SimModeCar.cpp Source\Vehicles\Multirotor\Flyin
  gPawn.cpp Source\Vehicles\Multirotor\MultirotorPawnEvents.cpp Source\Vehicles\Multirotor\MultirotorPawnSimApi.cpp Sou
  rce\Vehicles\Multirotor\SimModeWorldMultiRotor.cpp Source\WorldSimApi.cpp

  Logger.cpp
  NedTransform.cpp
  PawnSimApi.cpp
  PInvokeWrapper.cpp
  SimHUD.cpp
  SimModeBase.cpp
d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\worldsimapi.h(16): error C3668: 'WorldSimAp
i::resetImplementation': method with override specifier 'override' did not override any base class methods [D:\Git\Open
Source\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj]
d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\simmode\simmodebase.cpp(18): error C2259: '
WorldSimApi': cannot instantiate abstract class [D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\A
irsimWrapper.vcxproj]
  d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\simmode\simmodebase.cpp(18): note: due to
   following members:
  d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\simmode\simmodebase.cpp(18): note: 'void
  msr::airlib::WorldSimApiBase::reset(void)': is abstract
  d:\git\opensource\microsoft\airsim\airlib\include\api\worldsimapibase.hpp(29): note: see declaration of 'msr::airlib:
  :WorldSimApiBase::reset'
  SimModeWorldBase.cpp
  UnityImageCapture.cpp
  UnityDistanceSensor.cpp
  UnitySensorFactory.cpp
  UnityToAirSimCalls.cpp
  CarPawn.cpp
  CarPawnApi.cpp
  CarPawnSimApi.cpp
  SimModeCar.cpp
  FlyingPawn.cpp
  MultirotorPawnEvents.cpp
  MultirotorPawnSimApi.cpp
  SimModeWorldMultiRotor.cpp
  WorldSimApi.cpp
d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\worldsimapi.h(16): error C3668: 'WorldSimAp
i::resetImplementation': method with override specifier 'override' did not override any base class methods [D:\Git\Open
Source\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj]
Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj" (def
ault targets) -- FAILED.

Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metap
roj" (default targets) -- FAILED.

Done Building Project "D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper.sln" (Clean;Build target(s)
) -- FAILED.


Build FAILED.

"D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper.sln" (Clean;Build target) (1) ->
"D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj.metaproj" (default target)
(2:2) ->
"D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj" (default target) (5:2) ->
(ClCompile target) ->
  d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\worldsimapi.h(16): error C3668: 'WorldSim
Api::resetImplementation': method with override specifier 'override' did not override any base class methods [D:\Git\Op
enSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj]
  d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\simmode\simmodebase.cpp(18): error C2259:
 'WorldSimApi': cannot instantiate abstract class [D:\Git\OpenSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper
\AirsimWrapper.vcxproj]
  d:\git\opensource\microsoft\airsim\unity\airlibwrapper\airsimwrapper\source\worldsimapi.h(16): error C3668: 'WorldSim
Api::resetImplementation': method with override specifier 'override' did not override any base class methods [D:\Git\Op
enSource\Microsoft\AirSim\Unity\AirLibWrapper\AirsimWrapper\AirsimWrapper.vcxproj]

    0 Warning(s)
    3 Error(s)

Time Elapsed 00:01:57.94

D:\Git\OpenSource\Microsoft\AirSim\Unity>if ERRORLEVEL 1 goto :buildfailed

D:\Git\OpenSource\Microsoft\AirSim\Unity>echo(


D:\Git\OpenSource\Microsoft\AirSim\Unity>echo #### Build failed - see messages above.  1>&2
#### Build failed - see messages above.

D:\Git\OpenSource\Microsoft\AirSim\Unity>exit /b 1

D:\Git\OpenSource\Microsoft\AirSim\Unity>
```

--- a/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.cpp
+++ b/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.cpp
@@ -15,7 +15,7 @@ bool WorldSimApi::isPaused() const
 	return simmode_->isPaused();
 }
 
-void WorldSimApi::resetImplementation()
+void WorldSimApi::reset()
 {
 	simmode_->reset();
 }

--- a/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.h
+++ b/Unity/AirLibWrapper/AirsimWrapper/Source/WorldSimApi.h
@@ -13,7 +13,7 @@ public:
 	WorldSimApi(SimModeBase* simmode, std::string vehicle_name);
 	virtual ~WorldSimApi();
 	virtual bool isPaused() const override;
-	virtual void resetImplementation() override;
+	virtual void reset() override;
 	virtual void pause(bool is_paused) override;
 	virtual void continueForTime(double seconds) override;
         virtual void setTimeOfDay(bool is_enabled, const std::string& start_datetime, bool is_start_datetime_dst,

